### PR TITLE
Feature/jakarta el 8

### PR DESCRIPTION
**Pull request by dmaidaniuk**
PR for issue #809

### CODE REVIEW

File: src/main/java/com/hubspot/jinjava/el/tree/impl/Scanner.java [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+                }
+                return fixed(Symbol.LT);
+            case '>':
+                if (c2 == '=') {
+                    return fixed(Symbol.GE);
+                }
+                return fixed(Symbol.GT);
+            case '"':
+            case '\'': return nextString();
+            default: throw new IllegalArgumentException("Value " + c1 + " is not supported.");
```
Comment (jasmith-hs) [inline src/main/java/com/hubspot/jinjava/el/tree/impl/Scanner.java:386]: This here causes a very large amount of the unit tests to fail. As this PR is currently, over half of the unit tests are failing.

**Review by jasmith-hs — CHANGES_REQUESTED** [review]
I don't think we'd have a problem migrating off of the Odysseus implementation if it all works properly.
By the way, which repository did you get these new JUEL classes from, and is that repository still being maintained?

File: src/main/java/com/hubspot/jinjava/el/tree/impl/Scanner.java (hunk 1 above)
Comment (jasmith-hs) [reply src/main/java/com/hubspot/jinjava/el/tree/impl/Scanner.java:386]: Sorry, just realised you had marked this as a draft PR 3 days ago, I had originally looked over it when it was first opened.

Comment (dmaidaniuk) [reply src/main/java/com/hubspot/jinjava/el/tree/impl/Scanner.java]: > This here causes a very large amount of the unit tests to fail.

Fixed))

**Comment by dmaidaniuk** [comment]
> I don't think we'd have a problem migrating off of the Odysseus implementation if it all works properly. By the way, which repository did you get these new JUEL classes from, and is that repository still being maintained?

Well, I just built-in and upgrade Odysseus implementation according to EL 4 specification.

**Pull request closed without merging**